src/main/java/Duke.java

```diff
@@ -6,5 +8,82 @@ public static void main(String[] args) {
+                    break;
+                }
+                if (line.equals("list")) {
+                    System.out.println(dashes);
+                    System.out.println("Here are the tasks in your list:");
+
+                    for (int i = 0; i < Count; i++) {
+                        System.out.println((i + 1) + "." + List[i].getDescription());
+                    }
+                    System.out.println(dashes);
+                } else if (line.split(" ")[0].equals("done")) {
+                    int c = Integer.parseInt(line.split(" ")[1]);
+                    List[c - 1].markAsDone();
+                    System.out.println("Nice! I've marked this task as done: ");
+                    System.out.println((c) + "." + List[c - 1].getDescription());
+                } else {
+                    System.out.println(dashes);
+                    String desc;
+                    switch (line.split(" ")[0]) {
+                        case "todo":
+                            if (line.split(" ", 2).length < 2) {
+                                throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                            }
+                            desc = line.split(" ", 2)[1];
+                            List[Count] = new Todo(desc);
+                            break;
+                        case "deadline":
+                            if (line.split(" ", 2).length < 2) {
+                                throw new DukeException("☹ OOPS!!! The description of a deadline cannot be empty.");
+                            }
+                            desc = line.split(" ", 2)[1];
+                            List[Count] = new Deadline(desc.split("/by")[0], desc.split("/by")[1]);
+                            break;
+                        case "event":
+                            if (line.split(" ", 2).length < 2) {
+                                throw new DukeException("☹ OOPS!!! The description of a event cannot be empty.");
+                            }
+                            desc = line.split(" ", 2)[1];
+                            List[Count] = new Event(desc.split("/at")[0], desc.split("/at")[1]);
+                            break;
+                        default:
+                            throw new DukeException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                    }
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println("  " + List[Count].getDescription());
+                    System.out.println("Now you have " + (Count + 1) + " tasks in the list.");
+
+                    System.out.println(dashes);
+
+                    Count += 1;
+                }
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+                System.out.println(dashes);
+            }
+
+        }
+
+
+        String outro = "Bye. Hope to see you again soon!";
```

Comment:
Maybe can set it to a constant string since it will be used for every app execution